plugins/dynamodb/plugin.spec.yaml

```diff
@@ -47,24 +131,22 @@ actions:
         description: The table name to search
         type: string
         required: true
+        example: Table-name
       index:
         description: The index to use. If empty, defaults to a full Scan of the table
         type: string
         required: false
+        example: index-name
       params:
         description: The params to query with, as key/value pairs
         type: object
         required: false
+        example:
```

Comment:
Blank example, fill in?

Reply:
@r7-kszczepanskagorna can we get an example of params?